### ISSUE TITLE
Repaint problem of the clock area in system tray

### ISSUE THREAD

**Issue by Wirek78** [White-Tiger/T-Clock#113]
The repaint problem appears on _MS Windows XP Pro 32-bit EN_ on random occasions - not every time but often enough to be quite annoying (and quite easy to reproduce in my environment). My system format for time values is "HH:mm:ss", while _T-Clock Redux_ (v. 2.4.3 build 471) time format (in "Advanced clock format" field) is set to "HH:nn:ss" value. The problem affects only the seconds (":ss") part of the time format and value. I've just switched from _Stoic Joker's T-Clock 2010 Build 95_ where this particular problem didn't exist so it looks like some sort of regression issue. The problem doesn't exist on _MS Windows 7 Pro 32-bit PL_ since there's a different approach on hiding icons in the notification area of system tray.
Steps to reproduce the problematic situation are:
1. Click on the left arrow in the notification area to reveal hidden icons;
2. Part with seconds (":ss") of the displayed _T-Clock Redux_ value is hidden just like the format would change to "HH:nn" by itself and right shifted closer to the right edge of the screen;
3. Click anywhere to hide icons previously revealed in the notification area;
4. Part with seconds (":ss") of the clock in system tray area shows again (in correct position) but **sometimes** with black background;
5. Move mouse pointer over the seconds (":ss") part with black background and move mouse out of the area - the normal colour background is repainted.

The described situation is shown in the picture below ([http://i64.tinypic.com/12648z7.gif](http://i64.tinypic.com/12648z7.gif)).
![t-clock_redux](https://user-images.githubusercontent.com/31701622/30121603-6f6f0f5a-932c-11e7-8f6d-2a4a9f59ecdf.gif)

Summary of details:
- severity: random but quite often;
- OS: _MS Windows XP Pro 32-bit EN_;
- system time format: HH:mm:ss;
- system taskbar options: auto-hide;
- application: _T-Clock Redux version 2.4.3 **static** build 471_ (_Stoic Joker's T-Clock 2010 Build 95_ was not affected by this issue);
- application time format (also corresponding tab in options): HH:nn:ss;
- my "T-Clock.ini" settings:
```

[Clock]
Font=Trebuchet MS
ForeColor=-84082958
BackColor=-84082958
FontSize=11
Bold=0
Italic=0
FontQuality=5
ClockHeight=0
ClockWidth=0
LineHeight=0
VertPos=0
HorizPos=0
Angle=0
[Mouse]
01=8
11=1
21=5
DropFiles=0
DropFilesApp=
02=6
02Clip=""w" dddd "dnia" DATE "o godzinie" HH:nn:ss"
[Format]
Lf=0
AMPM=0
Hour12=0
Hour=1
Format=HH:nn:ss
Locale=1045
Year4=0
Year=1
Month=1
MonthS=0
Day=1
Weekday=1
Minute=1
Second=1
InternetTime=0
HourZero=0
Custom=1
AMsymbol=AM
PMsymbol=PM
CustomFormat=HH:nn:ss
[Main]
Ver=3
UpdR=0
UpdB=0
AlarmNum=0
Jihou=0
JihouFile=Clock.wav
JihouRepeat=0
JihouBlink=0
[Taskbar]
AlphaTaskbar=20
ClearTaskbar=0
[Tooltip]
Tooltip=DATE   dddd\n"dzień roku:" DOY
bCustom=1
[Timers]
NumberOfTimers=0
[Calendar]
ViewMonths=1
ViewMonthsPast=0
ShowDayOfYear=1
ShowWeekNums=1
CloseCalendar=1
CalendarTopMost=0
bCustom=1
[QuickyMenu]
DisplayProperties=0
AudioProperties=0
NetworkDrives=0
ExitWindows=0
QuickyMenu=1
[SNTP]
SaveLog=0
MessageBox=0
Server=pool.ntp.org
Sound=<  no sound  >
Server1=pool.ntp.org
Server2=europe.pool.ntp.org
Server3=north-america.pool.ntp.org
Server4=asia.pool.ntp.org
Server5=oceania.pool.ntp.org
Server6=south-america.pool.ntp.org
Server7=africa.pool.ntp.org
Server8=time1.google.com
Server9=time2.google.com
Server10=time3.google.com
Server11=time4.google.com
Server12=time.nist.gov
Server13=time.windows.com
ServerNum=13
[Desktop]
MonOffOnLock=0
Multimon=1
```

I hope this bug report appears helpful to you so you can trace the problem and fix it. I'm looking forward to next builds of this application.

**Comment by White-Tiger** [comment]
Well... if you need my version for some reason _(instead if Stoic's)_, or rather like to do a quick check if my suspicion is correct, please test your experience using a new "copy" of T-Clock with its own .ini file by using this pre-Win10 version: https://github.com/White-Tiger/T-Clock/releases/tag/v2.4.0%23384-rc

I believe it's those Win10 changes that cause these issues... I'm aware that the current behavior is quite hacky, yet Win10 forced me to go this ugly way...

**Comment by White-Tiger** [comment]
Now implemented in [v2.4.4](https://github.com/White-Tiger/T-Clock/releases/tag/v2.4.4%23492-rc)
